src/shims/ApiCompat.proj

```diff
@@ -26,6 +26,13 @@
     <_RunApiCompat Condition="'$(BuildAllConfigurations)' == 'true' and $(TargetGroup.StartsWith('uap'))">false</_RunApiCompat>
   </PropertyGroup>
 
+  <ItemGroup>
+    <!-- Run API compat against the following 1:1 assemblies but don't include them in the list that is used to generate facades -->
+    <NetFxReference Include="System.DirectoryServices" />
+    <NetFxReference Include="System.DirectoryServices.AccountManagement" />
+    <NetFxReference Include="System.DirectoryServices.Protocols" />
+  </ItemGroup>
+
   <!-- Run ApiCompat -->
   <Target Name="RunApiCompat"
           Condition="'$(_RunApiCompat)' == 'true'"
```

Comment:
We do run the netstandard compat checks all the time now but for the ones producing a baseline file we don't run them by default which is intentional to avoid files being updated in the source tree all the time. We run them manually from time to time.